### PR TITLE
Add linting, yarn lock, simplify dev script, automate with Travis

### PR DESCRIPTION
**Pull request by karlhorky**

### CODE REVIEW

File: dist/wizards.js [hunk 1]
```diff
@@ -126,12 +119,12 @@ function interactionWizard(args) {
         }
       },
       consumer: {
-        pattern: /^[a-zA-Z\-]+$/,
```
Comment (prumand) [inline dist/wizards.js]: are those regex changes discovered bugs?

Comment (karlhorky) [inline dist/wizards.js]: It's unnecessary escaping.

**Comment by prumand** [comment]
I added linting (in master) and travis in #17 Would have been a pain to merge.

**Comment by karlhorky** [comment]
Awesome, cheers @prumand !

**Pull request closed without merging**